Description: Do not install Japanese-language POD
 The included Japanese-language POD file in Test::LeakTrace::JA does not render
 properly, and breaks tools like pod2man, leading to warnings from lintian.
 This document is of limited usefulness from pod2man and thus is not installed
 in Debian.
Origin: vendor
Bug: https://rt.cpan.org/Ticket/Display.html?id=47493
Author: Ryan Niebur <ryanryan52@gmail.com>
Reviewed-by: Jonathan Yu <frequency@cpan.org>
--- a/Makefile.PL
+++ b/Makefile.PL
@@ -24,4 +24,5 @@
 		nytprof *.out
 		cover_db
 	)},
+	'MAN3PODS' => {'lib/Test/LeakTrace.pm' => '$(INST_MAN3DIR)/Test::LeakTrace.$(MAN3EXT)'},
 );
